rollup.config.js

```diff
@@ -1,7 +1,19 @@
+var pkg = require('./package.json');
 import buble from 'rollup-plugin-buble';
 
 export default {
 	entry: 'src/index.js',
 	plugins: [ buble() ],
-	external: [ 'path', 'minimatch' ]
+	external: [ 'path', 'minimatch' ],
+
+	targets: [
+		{
+				format: 'cjs',
```

Comment:
What's our convention with tabs here? I usually do 2 spaces but would rather stay consistent with the team. Merely a `.vimrc` setting for me.